app/src/app/components/Sidebar/index.tsx

```diff
@@ -53,16 +74,41 @@ function Sidebar() {
         </div>
         {isUserLoggedIn ? (
           <div className={`column ${styles.contentLinks} start`}>
-            <div className="row m-bottom-3">
+            <div className={`column m-bottom-3 ${styles.selectContainer}`}>
               <span className={styles.techTitle}>Tech:</span>
-              <select className={styles.techSelect} value={selectedTech} onChange={handleTechChange}>
-                <option value="all">Todas</option>
+              <div
+                className={cn(styles.boxTechSelected, {
+                  [`${styles.isOpen}`]: toogleOpen
```

Comment:
I think you don't need the interpolation 🤔 
```suggestion
                  [styles.isOpen]: toogleOpen
```
Does that work?